website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,219 @@
+New Detailed package in a more categorized manner for readme, dependencies, version and uplinks. 
+
+![alt text](./assets/detail-page.png "verdaccio@4 detail page")
+
+### <a id="package-sidebar"></a> Package sidebar
+
+Package sidebar includes most relevant information from package metadata. You can open an issue,see readme and download the package tarball. It is also clearly shows the package's minimum requirements on node and npm. 
+
+Also package sidebar shows *Author*, *Maintainers* and *Contributors* in different sections. When you click on person avatar, you'll be able to contact that person via email.
+
+## <a id="router-api-ui"></a> New Router APIs
+Till, verdaccio@3 we have hash router implementation on frontend application routes. We faced a lot of problem with hash router in readme section. Readmes also uses (#) hash for the heading tags and anchor elements. 
+
+In verdaccio@3, we migrated hash router to browser router with more cleaner look. (No more hashes in URLs).
+
+## <a id="unpublish-package-role"></a> Unpublish packages role
+
+Verdaccio@4 improves the package management by adding access layer for publish & unpublish. Now you can have restrictions to some of users for publish and unpublish. [Find out more](https://verdaccio.org/docs/en/packages )
+
+## <a id="disable-gravatar"></a> Disable Gravatar
+Verdaccio uses [Gravatar](https://en.gravatar.com) to show author, contributors and maintainers images. Now, gravatar support can be disabled from verdaccio `config.yaml`. 
+
+```
+web:
+  title: Verdaccio
+  gravatar: false
+```
+
+In order to be fully offline, The fallback support is a generic user face SVG based on base64.
+
+## <a id="new-npm-commands"></a> New commands
+
+We are really excited to add some npm cli commands to verdaccio. Now you can use `npm star`, and `npm profile`.
+
+### <a id="npm-star"></a> npm star
+
+Now a user can mark their favorite package. 
+
+```
+npm star [<package>..]
+```
+### <a id="npm-profile"></a> npm profile
+
+With npm profile an user change their profile settings. 
+
+*Note:* verdaccio does not support two factor authentication yet.
+
+```
+npm profile get [--json|--parseable] [<property>]
+npm profile set [--json|--parseable] <property> <value>
+npm profile set password
+```
+
+Check out more at [https://docs.npmjs.com/cli/profile](https://docs.npmjs.com/cli/profile)
+
+
+## <a id="jwt-token"></a> JWT token
+
+Verdaccio suppports JWT - [JSON Web Tokens](https://jwt.io/) for authentication process. Previos version of verdaccio used `AES` token generator. The new JWT token standardises the process and provides an additonal mechanism for token generation. Verdaccio 4 still support the `AES` token generator.
+
+[Click here for more information on new JWT tokens](https://medium.com/verdaccio/diving-into-jwt-support-for-verdaccio-4-88df2cf23ddc)
+
+## <a id="docker-improvements"></a> Docker improvements
+- https://github.com/verdaccio/verdaccio/pull/845
+@juanpicado
```

Comment:
:sweat_smile: I think this should be completed by @juanpicado

Reply:
@sergiohgz done below.